lib/user_parameter_sanitiser.rb

```diff
@@ -0,0 +1,44 @@
+class UserParameterSanitiser
+  def initialize(user_params:, current_user_role:, permitted_params_by_role: nil)
+    @user_params = user_params
+    @current_user_role = current_user_role
+    @permitted_params_by_role = permitted_params_by_role ||
+      default_permitted_params_by_role
```

Comment:
Rather than default the keyword parameter to `nil` and use `||` to set a fallback here, what about adding it to the params?

```ruby
def initialize(user_params:, current_user_role:, permitted_params_by_role: default_permitted_params_by_role)
```